metrics/integration.go

```diff
@@ -69,13 +70,13 @@ func RegisterCompletedSnapshot(conditiontype, reason string, startTime metav1.Ti
 		"reason": reason,
 	}
 
-	SnapshotConcurrentTotal.Sub(1)
+	SnapshotConcurrentTotal.WithLabelValues().Sub(1)
```

Comment:
If we don't specify any labels, why do we use GaugeVec? Ins't it just Gauge if we don't add "vectors" there then?

Reply:
Please see the reason below. Thanks